CIScripts/Test/PesterLogger/PesterLogger.ps1

```diff
@@ -59,3 +91,21 @@ function Write-Log {
         Write-Host @Args
     }
 }
+
+class LogItem {
+    [String] $Timestamp
+    [String] $Tag
+    [String] $Message
+}
+
+function ConvertTo-LogItem {
+    Param([Parameter(ValueFromPipeline, Mandatory=$true)] $Line)
+    Process {
+        $Timestamp, $Tag, $Message = $Line.Split("|", 3)
```

Comment:
Maybe we should add comments with example log lines (with and without a timestamp)?

Reply:
"The tests are examples"

Reply:
We shouldn't have an example of specific formatting, as mentioned in my earlier comment. Such test would be fragile and pointless.

If we had a complex regex here, and had 10 test cases to see if it matches correctly, I would still recommend having a "happy path example" just next to the regex. `This regex matches strings like asdfasdf`.

`ConvertTo-LogItem` exists only because we use actually files on TestDrive, instead of having a nice interface for the output method (we don't have something like the Emitter in Shelly, that can output to a vector or to stdout). `ConvertTo-LogItem` is pretty much a workaround for this fact. But it's a good workaround for now. 

However, trying to write tests that are coupled to specific formatting even more would move as away from the 'perfect' scenario of having replaceable output and formatting methods. We already have to rework `ConvertTo-LogItem` function every time we add a new tagging method (e.g. showing `timestamp | hostname | logname | msg`. Or `| priority |`. etc.